### PR TITLE
Exchange Maven Facade API version 3 documentation

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
Exchange Maven Facade API version 3 documentation

### CODE REVIEW

File: api-docs/2-Getting Started.md [hunk 1]
```diff
@@ -128,9 +128,9 @@ This example publishes a `custom` asset.
     </build>
 ```
 
-##### Requirements:
```
Comment (LucasLudueno) [inline api-docs/2-Getting Started.md:131]: what about replacing it with `Consideration`? or something like `Common error`? 🤔

**Review by LucasLudueno — APPROVED** [review]

File: api-docs/2-Getting Started.md (hunk 1 above)
Comment (wwarner-salesforce) [reply api-docs/2-Getting Started.md:131]: Good idea. How about "Troubleshooting"?

**Review by LucasLudueno — APPROVED** [review]

File: api-docs/2-Getting Started.md (hunk 1 above)
Comment (LucasLudueno) [reply api-docs/2-Getting Started.md:131]: I like it

**Review by LucasLudueno — APPROVED** [review]

**Comment by wwarner-salesforce** [comment]
https://anypoint.mulesoft.com/exchange/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/exchange-maven-facade-api-http/

**Review by LucasLudueno — APPROVED** [review]

**Comment by wwarner-salesforce** [comment]
Copied changes to Maven Facade API docs on Exchange.

**Pull request closed without merging**